cardano-node/src/Cardano/CLI/Tx.hs

```diff
@@ -0,0 +1,114 @@
+{-# OPTIONS_GHC -Wno-all-missed-specialisations #-}
+
+module Cardano.CLI.Tx
```

Comment:
Like wise I also think `Cardano.CLI.Tx` and `Cardano.CLI.Tx.Submission` should be condensed to one module. Having a module for a single function is overkill IMO.

Reply:
We are going to have `Cardano.Tx.Generator` soon -- this is a preparation for that.